backend/python/app/services/implementations/story_service.py

```diff
@@ -894,6 +897,30 @@ def get_story_translation_statistics(self):
             num_translations_completed=num_translations_completed,
         )
 
+    def soft_delete_story(self, id):
+        try:
+            story = StoryAll.query.get(id)
+            for content in story.contents:
+                content.is_deleted = True
+
+            story.is_deleted = True
+            story.translated_languages = None
```

Comment:
remove this line, its not necessary to empty the array and we'd like to preserve information of the stories